packages/pluggableWidgets/datagrid-web/src/utils/__tests__/settings.spec.ts

```diff
@@ -170,6 +170,61 @@ describe("useSettings Hook", () => {
             ])
         );
     });
+
+    it("doesnt change the settings when same properties are applied", () => {
```

Comment:
Try to add some more tests to cover more flows.

Reply:
Done